frontend/pages/projects/new.js

```diff
@@ -45,14 +47,26 @@ const Upload = ({ user, csrf }) => {
       // In the case of failing to create the repo, i.e., it already exits.
 
       // It should display a modal asking for overwriting the existing project
+      setOpen(true)
       console.error('Repo already exists!')
     } else {
       loadFiles(files, tempProjectName)
       await push(`/projects/update/${user.login}/${tempProjectName}?create=true`)
     }
   }
 
-  return <DropZone onDrop={onDrop} />
+  return (
+    <>
+      <DropZone onDrop={onDrop} />
+      <Modal
+        closeIcon
+        header="Heads up!"
+        content="You have an existing project with the same name, do you want to overwrite it?"
+        open={open}
+        onClose={() => setOpen(false)}
+      />
+    </>
+  )
```

Comment:
I think we shouldn't offer to overwrite a project when uploading a single file, rather the options should be

1. Update an existing project
2. Create a new project with a different name